File: em Vídeo/POO [60 Horas]/Java POO [40 Horas]/Aula 10/Aluno.java
public class Aluno extends Pessoa {
    private int matr;
    private String curso;

    public int getMatr() {
        return matr;
    }

    public void setMatr(int matr) {
        this.matr = matr;
    }

    public String getCurso() {
        return curso;
    }

    public void setCurso(String curso) {
        this.curso = curso;
    }

    public void cancelarMatr() {
        System.out.println("Matrícula será cancelada");
        this.setMatr(0);
    }

}

File: CursoEmVideo/POO [60 Horas]/PHP POO [40 Horas]/Aula 10/Aluno.php
<?php
    require_once 'Pessoa.php';
    class Aluno extends Pessoa {
        private $matricula;
        private $curso;

        function getMatricula() {
            return $this -> matricula;
        }

        function setMatricula($matricula) {
            $this -> matricula = $matricula;
        }

        function getCurso() {
            return $this -> curso;
        }

        function setCurso($curso) {
            $this -> curso = $curso;
        }

        function cancelarMatricula() {
            print("Matrícula será cancelada.<br>");
            $this -> setMatricula(0);
        }
    
        function pagarMensalidade() {
            print("Pagando mensalidade do aluno ".$this->getNome()."<br>");
        }

    }
